sysapi/sysapi_util/CommandUtil.c

```diff
@@ -159,16 +158,11 @@ TSS2_RC CommonComplete(_TSS2_SYS_CONTEXT_BLOB *ctx)
         rval = Tss2_MU_UINT32_Unmarshal(ctx->cmdBuffer,
```

Comment:
If we are not saving anything here then the comment above is irrelevant. Please remove

Reply:
Changed to "Skipping over"